components/x-gift-article/src/CopyConfirmation.jsx

```diff
@@ -9,14 +9,18 @@ const confirmationClassNames = [
 	styles['copy-confirmation']
 ].join(' ')
 
-export default ({ hideCopyConfirmation }) => (
+export default ({ hideCopyConfirmation, isTestVariant }) => (
 	<div className={confirmationClassNames} role="alert">
 		<div className={styles['o-message__container']}>
 			<div className={styles['o-message__content']}>
 				<p className={styles['o-message__content-main']}>
-					<span className={styles['o-message__content-highlight']}>
-						The link has been copied to your clipboard
-					</span>
+					{isTestVariant ? (
```

Comment:
I would expect if this is an A/B test to return either 'control' or 'variant' so I think this would need to be `isTestVariant === 'variant'`

Reply:
Ah, well spotted - I haven't worked much with flags before, I assumed it would be "flag - true / false"